test/test_config.py

```diff
@@ -69,7 +69,34 @@ def test_load_dict(self):
         c = ConfigDict()
         c.load_dict({key: {'subkey': 'value'}})
         self.assertEqual('value', c[key + '.subkey'])
-   
+
+    @classmethod
+    def setUpClass(self):
+        # create config file
+        from configparser import ConfigParser
+        config = ConfigParser()
+        # two special section
+        config['DEFAULT'] = {'interval': '45'}
+        config['bottle'] = {'port': '8080'}
+        # ordinary section
+        config['compression'] = {'status': 'single'}
+        with open('example.ini', 'w') as config_file:
```

Comment:
I would use a `tempfile` for this, https://docs.python.org/2/library/tempfile.html#tempfile.mkstemp and use [`TestCase.addCleanup`](https://docs.python.org/2/library/unittest.html#unittest.TestCase.addCleanup) to ensure it is removed when finished. If you use `addCleanup`, you won't need `tearDownClass`.

```python
import functools
import os
import tempfile

...
        # changed from config_file to self.config_file
        self.config_file = tempfile.mkstemp(suffix='.example.ini')
        self.addCleanup(functools.partial(os.remove, self.config_file.name))

        # down in test_load_config()
        c.load_config(self.config_file.name)
```